client/containers/DashboardSubmissions/SubmissionItems.tsx

```diff
@@ -76,7 +77,12 @@ const SubmissionItems = (props: Props) => {
 
 	const renderPubs = () => {
 		return pubs.map((pub) => (
-			<PubOverviewRow pub={pub} key={pub.id} leftIconElement="manually-entered-data" />
+			<PubOverviewRow
+				pub={pub}
+				key={pub.id}
+				leftIconElement="manually-entered-data"
+				isSubmission={true}
```

Comment:
You can just write `isSubmission` instead of `isSubmission={true}` here (if you like)

Reply:
i do like explicit declaration here